cmd/format-fs.go

```diff
@@ -333,11 +333,17 @@ func formatFSFixDeploymentID(fsFormatPath string) error {
 		return nil
 	}
 
+	attempts := 0
 	for {
 		wlk, err := lock.TryLockedOpenFile(fsFormatPath, os.O_RDWR, 0)
 		if err == lock.ErrAlreadyLocked {
-			// Lock already present, sleep and attempt again.
+			// Lock already present, sleep and attempt again
+			// until for about 5 seconds.
 			time.Sleep(100 * time.Millisecond)
+			attempts++
+			if attempts == 50 {
+				return fmt.Errorf("%s: %s", fsFormatPath, err.Error())
```

Comment:
Looks like we only need to show a better error message for this and use your suggestion @kannappanr before approving this PR

Reply:
done